features/general/file-manager.md

```diff
@@ -0,0 +1,105 @@
+# File Manager of ERP.net
+
+## Overview
+
+Oftentimes users have to search through complex directories to find a file they are looking for. 
+Especially when the file is **not associated** to a **specific document or record** in the ERP system, buth is a rather more **generic document** for business use, such as documentation regarding training courses, internal or external regulations, company practices, handbooks, and so on.  
+
+Users tend to keep such files in an external storage, like local drives or cloud platforms. But this often leaves a gap between navigating through internal ERP entities and external file storage.  
```

Comment:
Start new sentences on new line.

Reply:
Moved the sentence to a new line.
Also checked for other identical errors and corrected throughout the whole document.